ArduPlane/quadplane.cpp

```diff
@@ -1410,7 +1410,16 @@ void SLT_Transition::update()
         (quadplane.transition_failure > 0) &&
         ((now - transition_start_ms) > ((uint32_t)quadplane.transition_failure * 1000))) {
         gcs().send_text(MAV_SEVERITY_CRITICAL, "Transition failed, exceeded time limit");
-        plane.set_mode(plane.mode_qland, ModeReason::VTOL_FAILED_TRANSITION);
+        // for STL if option is set and ground speed> 1/2 arspd_fbw_min, then complete transition, otherwise QLAND.
+        // all others will immediately transition
+        if (quadplane.options & QuadPlane::OPTION_TRANS_FAIL_TO_FW) {
+            if (!plane.quadplane.tailsitter.enabled() && !plane.quadplane.tiltrotor.enabled()
```

Comment:
```suggestion
            if (!plane.quadplane.tiltrotor.enabled()
```
Don't need this tailsitter check.